fiftyone/operators/registry.py

```diff
@@ -40,6 +45,21 @@ def list_operators(enabled=True):
     return registry.list_operators(include_builtin=enabled != False)
 
 
+def operator_exists(operator_uri, enabled=True):
```

Comment:
I don't know why we need this. There already is `registry.operator_exists`

Reply:
You needed it in https://github.com/voxel51/fiftyone/pull/3692. You wrote:

```py
if op.operator in foo.list_operators():
```

which would be nicer to read as:

```py
if foo.operator_exists(op.operator):
```